delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/rejection/Rejection.scala

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.bluebrain.nexus.delta.sourcing.rejection
+
+abstract class Rejection extends Exception with Product with Serializable {
+
+  def reason: String
```

Comment:
I think the message in exception should probably be this reason, perhaps even reason should be an argument. It's definitely worth thinking about this now before everything is using it and it's hard to change

Reply:
The `reason` is currently brought up all the way to the API to serialize the errors so it is difficult to change `reason` for `message`